### ISSUE TITLE
viewer.color_by is slow for large molecules

### ISSUE THREAD

**Issue by avirshup** [Autodesk/notebook-molecular-visualization#39]
This actually looks cool, although it's probably not what we want :)

**To reproduce:**
In a notebook, execute this cell:
```python
mol = mdt.from_pdb('3aid')
drug = mol.chains['A'].get_ligand()
viewer = mol.draw(display=True)
viewer.stick()
```

Then this one:
```python
viewer.color_by(lambda atom:atom.distance(drug))
```

In the latest version, the colors draw in 1-at-a-time, which takes a while:
![color_by_slow](https://cloud.githubusercontent.com/assets/9388007/19950966/459b8c28-a118-11e6-96ec-b6df628b3e4f.gif)

In 0.7.3, the color comes in all-at-once after thinking for a second or two:
![color_by_fast](https://cloud.githubusercontent.com/assets/9388007/19951029/ac41e684-a118-11e6-985d-ba5ceed3ca89.gif)

**Comment by justinmc** [comment]
Were you running the latest code that includes my performance PR https://github.com/Autodesk/notebook-molecular-visualization/pull/27 when you saw this?  Just making sure.

**Comment by avirshup** [comment]
Yeah - it's all from the latest master (fb2eee08faca347444dc7bbf85de2d8792ccd6f2)

**Comment by justinmc** [comment]
Solution here: https://github.com/Autodesk/notebook-molecular-visualization/pull/42

It was syncing the data over the network 1 color at a time, which was very slow AND caused a rerender for reach color.  Now we do it all at once.  The frontend still has to call 3dmol's `setStyle` for each color, which is a lot, but it happens fast enough and with no pop-in.
